default.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "rebaseWhen": "behind-base-branch",
+    "dependencyDashboard": false,
+    "branchPrefix": "renovate-",
+    "constraints": {
+        "python": "3.8.10"
+    },
+    "pip-compile": {
+        "fileMatch": [
+            "(^|/)requirements\\.in$",
+            "(^|/)requirements-fmt\\.in$",
+            "(^|/)requirements-lint\\.in$",
+            "(^|/)requirements-unit\\.in$",
+            "(^|/)requirements-integration\\.in$",
+            "(^|/)requirements.*\\.in$"
+        ],
+        "lockFileMaintenance": {
+            "enabled": true,
+            "schedule": [
+                "on the 2nd and 4th day instance on sunday after 9pm"
```

Comment:
can we omit 9pm ?

like anytime on sunday?

I read that renovate recommends to put the smallest limit on execution time as possible.